### PR TITLE
{bio}[dummy] Transrate 1.0.3 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic68 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b13b5613fc1eb0bb91da2abf1f0c3143 for a full test report.

File: easybuild/easyconfigs/t/Transrate/Transrate-1.0.3-foss-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+sources = ['transrate-%(version)s-linux-x86_64.tar.gz']
+source_urls = ['https://bintray.com/artifact/download/blahah/generic']
+
+checksums = ['68d034ecd7012f1d3d505a2edd820c1155cd8b64d2acbf2ac833f30d3800141b']
+
+dependencies = [('BLAST+', '2.6.0', '-Python-2.7.13')]
+
+sanity_check_paths = {
+    'files': ['transrate', 'bin/bam-read', 'bin/salmon', 'bin/snap-aligner', 'lib/libruby.so'],
```
Comment (boegel) [inline easybuild/easyconfigs/t/Transrate/Transrate-1.0.3-foss-2017a.eb]: @hajgato please use `SHLIB_EXT`:

```python
'lib/libruby.%s' % SHLIB_EXT
```

Comment (hajgato) [reply easybuild/easyconfigs/t/Transrate/Transrate-1.0.3-foss-2017a.eb]: @boegel I did it.

**Comment by hajgato** [comment]
Apparently, the bundled `Salmon` fails in multiple cases, I have to upgrade to 0.8.2
See: https://github.com/blahah/transrate/issues/210  It is corrected in version `1.0.4` but it is not yet released.

File: easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-foss-2017a-Python-2.7.13.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
+
+patches = [
+    'BLAST+-2.2.30_ictce-fixes.patch',
+    'BLAST+-%(version)s_fix-make-install.patch',
+]
+
+checksums = ['0510e1d607d0fb4389eca50d434d5a0be787423b6850b3a4f315abc2ef19c996']
```
Comment (boegel) [inline easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-foss-2017a-Python-2.7.13.eb]: @hajgato Please use `--include-checksums` to inject checksums for both `sources` & `patches`...

Comment (hajgato) [reply easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-foss-2017a-Python-2.7.13.eb]: @boegel Done!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/db0fc69d01f7026b189ddec913a3a52f for a full test report.

**Comment by boegel** [comment]
(close + re-open to get latest checks in Travis)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2106.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6369598ae625df34bbb2c86f7c673502 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**